src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -12588,51 +12582,44 @@ void func_8084FF7C(Player* this) {
     }
 }
 
-#ifdef NON_MATCHING
-// regalloc differences
 void func_8085002C(Player* this) {
-    s32 pad;
+    s16 v0 = D_80858AC8.unk_08, v1 = D_80858AC8.unk_06;
```

Comment:
for now we are avoiding sameline declared incase we run into a situation where we need to flip stack order in the future. unless its required here?